### PR TITLE
fix glmmTMB and response_data2 tests

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
The 4 failing tests also failed locally due to the addition of the dispersion parameter and some changes in the cbind() return. Must now investigate the brms

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
Hum, brms testing fails with the following:

```
> library(testthat)
> library(insight)
> 
> if (length(strsplit(packageDescription("insight")$Version, "\\.")[[1]]) > 3) {
+   Sys.setenv("RunAllinsightTests" = "yes")
+ } else {
+   Sys.setenv("RunAllinsightTests" = "no")
+ }
> 
> test_check("insight")
── 1. Error: (unknown) (@test-brms.R#17)  ──────────────────────────────────────
StanHeaders version is ahead of rstan version; see https://github.com/stan-dev/rstan/wiki/RStan-Transition-Periods
1: brm(count ~ log_Age_c + log_Base4_c * Trt + (1 | patient), data = epilepsy, family = poisson(), 
       prior = bprior1, chains = 1, iter = 500) at testthat/test-brms.R:17
2: eval_silent(run(rstan::stan_model, stan_model_args), silent = silence_stan_model, 
       type = "message")
3: utils::capture.output(out <- eval(expr, envir), type = type, ...)
4: evalVis(expr)
5: withVisible(eval(expr, pf))
6: eval(expr, pf)
7: eval(expr, pf)
8: eval(expr, envir)
9: eval(expr, envir)
10: run(rstan::stan_model, stan_model_args)
11: eval2(call, envir = args, enclos = parent.frame())
12: eval(expr, envir, ...)
13: eval(expr, envir, ...)
14: .fun(model_code = .x1)
15: stop("StanHeaders version is ahead of rstan version; ", "see https://github.com/stan-dev/rstan/wiki/RStan-Transition-Periods")
══ testthat results  ═══════════════════════════════════════════════════════════
OK: 257 SKIPPED: 1 FAILED: 1
1. Error: (unknown) (@test-brms.R#17) 
Error: testthat unit tests failed
```

**Comment by DominiqueMakowski** [comment]
Best is to skip it on travis. Will try to do that.

**Comment by DominiqueMakowski** [comment]
There is the [skip_on_travis()](https://www.rdocumentation.org/packages/testthat/versions/0.11.0/topics/skip) function, but it seems to work only in test_that blocks. Unfortunately, the error is generated outside the test_that blocks, during model fitting. 

I tried to put the model fitting inside a test_that block, but then the others blocks fail because the models are not found.

**Comment by DominiqueMakowski** [comment]
Asked for help [here](https://stackoverflow.com/questions/54550233/skip-a-specific-test-file-on-travis-in-r) 😅

**Comment by strengejacke** [comment]
I think we can check for environment variables, see
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

File: tests/testthat/test-brms.R [hunk 1]
```diff
@@ -1,144 +1,144 @@
...
-#           zero_inflated = c("child", "camper"),
-#           zero_inflated_random = "persons"
-#         ))
-#
-#       expect_identical(find_terms(m4, flatten = TRUE), c("count", "child", "camper", "persons"))
-#     })
-#
-#   }
-# }
+.runThisTest <- Sys.getenv("RunAllinsightTests") == "yes"
```
Comment (strengejacke) [inline tests/testthat/test-brms.R]: Maybe we can add a check for Travis environment here, too?

Comment (DominiqueMakowski) [reply tests/testthat/test-brms.R]: yes but I don't know how 😮

Comment (DominiqueMakowski) [reply tests/testthat/test-brms.R]: The last one work though 🎉

**Comment by DominiqueMakowski** [comment]
Guess we can merge this and then we can try to fix the few remaining tests in brms and try to increase the test coverage a bit (seems that some functions are not tested at all).

Unless you are not convinced by the fact of keeping travis and codecov :)

**Comment by strengejacke** [comment]
I'll look into the coverage later and see which functions are not covered by tests yet.

**Pull request merged**